kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/utils/MetadataUtils.java

```diff
@@ -335,7 +338,10 @@ private static void createTopicIfNotExist(final KafkaServiceConfiguration conf,
         if (partitioned) {
             log.info("Creating partitioned topic {} (with {} partitions) if it does not exist", topic, numPartitions);
             try {
-                admin.topics().createPartitionedTopic(topic, numPartitions);
+                Map<String, String> properties =
```

Comment:
Do we need to add this property for creating non-partitioned topic?

Reply:
Yes, added.